mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -341,6 +418,14 @@ class Completions(
               val include =
                 !isUninterestingSymbol(sym) &&
                   isNotLocalForwardReference(sym)
+                  && (!sym.toString.startsWith(
+                    "object "
+                  ) || (!isTypePosition && !isNewPosition))
+                  && (!isTypePosition || !sym.toString.startsWith("method "))
+                  && (!isInstantiationOrMethodCallPos || (!sym.toString
+                    .startsWith("class ") && !sym.toString.startsWith(
+                    "trait "
```

Comment:
Same here, use a more robust way like flags. You can find what makes those kind strings in `Symbol.toString` here https://github.com/lampepfl/dotty/blob/b1afa225353100fc47092edf923af2b946ff88cd/compiler/src/dotty/tools/dotc/core/SymDenotations.scala#L1491-L1500
(you can navigate to here with a few hops by find definitions on `Symbol.toString`

Reply:
Thank you 🙏 super helpful